wiki/Tournaments/GSTLIVE/22/en.md

```diff
@@ -0,0 +1,115 @@
+
+## Links
+
+- **[Newspost]()**
+- [Discord](https://discord.gg/3YXF2UCEYs)
+- [Website](https://gstlive.org)
+- [Livestream](https://www.twitch.tv/osusg)
+- [Challonge bracket](https://challonge.com/gstlive2022)
+
+## Ruleset
+
+### General Rules
+
+1. GST:LIVE 2022 is a **National, Single Elimination, 2v2 tournament** with a team size of **2 players**.
+
+2. All matches will be played on **NoFail, Score v2 and Team VS**, with certain exceptions.
+
+3. The **top 8** teams from Qualifiers will move on to the Bracket Stage.
+
+4. For each map that doesn't require the Hidden mod, players may choose to play said maps with Hidden, with those scores divided by 1.06.
+
+### Registration
+
+1. Registration will be through the [GST:LIVE website](https://gstlive.org).
+
+2. There is Badge Weighted Seeding (BWS) in effect; the formula is rank^(0.9937^(badges^2)).
+
+3. Average team BWS rank must be #2000 or lower.
+
+### Qualifiers regulations
+
+1. All teams will play through the Qualifiers pool, consisting of **9 maps**, in order from A1 to A9.
+
+2. Each team must play through all the maps; failure to do so will result in disqualification.
+
+3. If a player disconnects or encounters any technical issues within the first **30 seconds** of a map, they are allowed to replay that particular map after the lobby is done with all the remaining maps.
+
+4. Only the top 8 teams will proceed to the bracket stage. Seeding in Qualifiers will be determined by the **sum of z-score of each map**.
+
+### Bracket stage regulations
+
+1. The Bracket Stage will be held **offline** on 17th December 2022 at AEF Online 2022. **By signing up, you agree that you are free and will be showing up to the event should you place top 8 in Qualifiers.** Should there be an emergency such that you are unable to make the event, please let The Staff know by messaging **megumic** (ue#0001) or **\_gt** (gt#8668) on Discord.
+
+2. If a player is not present at the time of their match, the match will be forfeited. **The Staff reserve the right to ban absent players for future iterations of GST and GST:LIVE.**
+
+3. There will be 2 pools; the Quarterfinals pool will be **BO7**, and the Semifinals/Finals pool will be **BO9**.
+
+### Match procedure
+
+1. Higher roll gets **second ban first pick**. 
+
+2. Each team will be given a maximum of **90 seconds** for every action (banning/picking a map). Failure to decide before the time limit will result in the action given to the opponent.
+
+3. After a beatmap is selected, teams will have a maximum of **90 seconds** to ready up.
+
+4. For each map that doesn't require the hidden mod, players may choose to play said maps with hidden, with those scores divided by 1.06. As such, you are required to co-operate by putting on the required mods for each map as the multiplayer lobby will virtually permanently have Freemod enabled.
+
+5. In the case that both teams reach match point, there will be **3 maps** left.
```

Comment:
what 3 maps? is this random? are there a tiebreaker pool with 3 tiebreakers?

Reply:
will make it clearer, but basically the mappool is designed such that when both teams reach match points there will be 3 maps left in the mappool (bo7 mappool has 9 maps so after 6 points, at 3-3, there are 3 maps left)